overlord/state/state.go

```diff
@@ -80,6 +80,9 @@ const (
 	RestartUnset RestartType = iota
 	RestartDaemon
 	RestartSystem
+	// RestartSystemImmediate is like RestartSystem but action is immediate,
+	// without unnecessary delay
+	RestartSystemImmediate
```

Comment:
this is slightly ungrammatical,  it would have to be RestartSystemImmediately.  Maybe we should just call it RestartSystemNow ?

Reply:
Renamed